<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnyActionSelectDialog</class>
 <widget class="QDialog" name="AnyActionSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Area</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pbArea01">
        <property name="text">
         <string>1</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bgArea</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbArea02">
        <property name="text">
         <string>2</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bgArea</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbArea03">
        <property name="text">
         <string>3</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bgArea</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbArea04">
        <property name="text">
         <string>4</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bgArea</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbArea05">
        <property name="text">
         <string>5</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bgArea</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbArea06">
        <property name="text">
         <string>6</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bgArea</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Map</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pbMap01">
        <property name="text">
         <string>1</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bgMap</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMap02">
        <property name="text">
         <string>2</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bgMap</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMap03">
        <property name="text">
         <string>3</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bgMap</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMap04">
        <property name="text">
         <string>4</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bgMap</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMap05">
        <property name="text">
         <string>5</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bgMap</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMap06">
        <property name="text">
         <string>6</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bgMap</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>Setting</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="leCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AnyActionSelectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AnyActionSelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="bgMap"/>
  <buttongroup name="bgArea"/>
 </buttongroups>
</ui>
